src/components/Auth/UsersCard/table/UsersTable.tsx

```diff
@@ -23,44 +26,82 @@ import {
 import { AuthUser } from '../../types';
 import EditUserDialog from '../../UserFormDialog/EditUserDialog';
 import { AuthZeroState } from './AuthZeroState';
+import { confirmDeleteUser } from './confirmDeleteUser';
 import { NoResults } from './NoResults';
+import { ProviderCell } from './ProviderCell';
+import styles from './UsersTable.module.scss';
 
 export type UsersTableProps = PropsFromDispatch & PropsFromStore;
 
 interface UserTableRowProps {
   user: AuthUser;
   onEditUser: () => void;
-  onSetUserEnabled: () => void;
+  setUserDisabled: () => void;
```

Comment:
This should either take a boolean, or be renamed to `toggleUserDisabled`.

Reply:
It's a toggle! Renamed